conjure-java-core/src/integrationInput/java/com/palantir/product/EteServiceEndpoints.java

```diff
@@ -697,7 +697,7 @@ public void handleRequest(HttpServerExchange exchange) throws IOException {
             AuthHeader authHeader = runtime.auth().header(exchange);
             Map<String, Deque<String>> queryParams = exchange.getQueryParameters();
             String queryParamNameRaw =
-                    runtime.serde().deserializeString(queryParams.get("queryParamName"));
+                    runtime.serde().plain().deserializeString(queryParams.get("queryParamName"));
```

Comment:
I think `runtime.plain()` is a cleaner here

Reply:
@rubenfiszel thoughts?

Reply:
Perhaps it works better if I also rename `SerDe` to `BodySerDe` and `UndertowRuntime.body()`?

Reply:
Yes, either one or the other but SerDe and PlainSerDe don't make sense to coexist.

Reply:
Yep, updated to `PlainSerDe` and `BodySerDe`